src/mn_setup.c

```diff
@@ -3847,9 +3847,9 @@ void MN_SetupResetMenu(void)
 
 void MN_BindMenuVariables(void)
 {
-    BIND_NUM(resolution_scale, 0, 0, UL, "Resolution scale menu index");
+    BIND_NUM(resolution_scale, 0, 0, UL, "Position of lever on the resolution scale slider");
```

Comment:
`Position of resolution scale slider`

Reply:
Not too sure about this one. @fabiangreffrath, @rfomin?

Reply:
It's an internal configuration item that shouldn't be user editable at all, it's a restriction of the menu/config code. I like the old "Resolution scale menu index", or no help line at all.

Reply:
Whatever description we go with, should we append something like `(do not modify)` to it?

Reply:
> Whatever description we go with, should we append something like (do not modify) to it?

If you want. There's also an internal `midi_player_menu` as well.

Fun fact: `complevel` in PrBoom was also an internal menu index, and now we're stuck with those numbers forever 😄.

Reply:
"Position of resolution scale slider (do not modify)"